applications/openshift/general/alert_receiver_configured/rule.yml

```diff
@@ -31,7 +31,7 @@ ocil_clause: 'Alert receiver is not configured'
 
 ocil: |-
     Run the following command to see if alert receiver has been configured to send alerts to a orgzinational defined external system:
```

Comment:
Since we're fixing the OCIL would you mind also fixing orgzinational to organizational ?

Reply:
Yes - I saw that. Thanks for reminding me.